modules/battenberg/1.1/config/default.yaml

```diff
@@ -10,18 +10,15 @@ lcr-modules:
 
         scratch_subdirectories: []
 
-        reference_path:
-            hg38: "__UPDATE__"
-            grch37: "__UPDATE__"
-
         options:
             #update and add/remove these lines as needed for the reference genomes being used. 
             chr_prefixed_reference:
                 hg38: " --chr_prefixed_genome "
                 grch37: " "
 
         conda_envs:
-            battenberg: "{MODSDIR}/envs/battenberg-1.0.yaml"
+            battenberg: "{MODSDIR}/envs/battenberg-1.1.yaml"
```

Comment:
Where is this file? I don't see it in the PR.

Reply:
The file was already present, I just linked it

Reply:
The version 1.1 should remain as is after you update it to 1.2, please make sure 1.1 corresponds to the one on master